MonoGame.Framework/Media/Song.NVorbis.cs

```diff
@@ -4,19 +4,29 @@
 
 using System;
 using System.IO;
-using Microsoft.Xna.Framework.Audio;
-using Tao.Sdl;
+using MonoGame.Utilities;
+using OpenTK.Audio;
 
 namespace Microsoft.Xna.Framework.Media
 {
     public sealed partial class Song : IEquatable<Song>, IDisposable
     {
-        private IntPtr _audioData;
-        private int _volume = 128; // in SDL units from 0 to 128
+        private AudioContext acontext;
+        private OggStreamer oggstreamer;
```

Comment:
This is so funny... I look at your comment, I look at my code, and I remember, I am creating audiocontext here, but soundcontroller creates it as well... I think I may know what I need to do :)

> How about MediaPlayer instead since it is a bit more closely related to music/audio?

Sure, any place that is instanced once is fine. I was going to put it in OpenTKPlatform only because soundcontroller was created there.